tests/test_usm_type.py

```diff
@@ -1469,6 +1469,19 @@ def test_histogram(usm_type_v, usm_type_w):
     assert edges.usm_type == du.get_coerced_usm_type([usm_type_v, usm_type_w])
 
 
+@pytest.mark.parametrize("usm_type_v", list_of_usm_types, ids=list_of_usm_types)
+@pytest.mark.parametrize("usm_type_w", list_of_usm_types, ids=list_of_usm_types)
+def test_bincount(usm_type_v, usm_type_w):
```

Comment:
Please add a test to `test_sycl_queue.py` also.

Reply:
Added